cmd/tag-remove.go

```diff
@@ -0,0 +1,95 @@
+/*
+ * MinIO Client (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"errors"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/console"
+)
+
+var tagRemoveCmd = cli.Command{
+	Name:   "remove",
+	Usage:  "remove tags assigned to the object",
+	Action: mainRemoveTag,
+	Before: setGlobalsFromContext,
+	Flags:  globalFlags,
+	CustomHelpTemplate: `Name:
+	{{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} [COMMAND FLAGS] TARGET
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+DESCRIPTION:
+   Delete object tags previously assigned (if any) to target.
+
+EXAMPLES:
+  1. Delete the tag values to testobject in the bucket testbucket on alias s3.
+     {{.Prompt}} {{.HelpName}} s3/testbucket/testobject
+
+`,
+}
+
+func checkRemoveTagSyntax(ctx *cli.Context) {
+	if len(ctx.Args()) != 1 {
+		cli.ShowCommandHelp(ctx, "remove")
+		os.Exit(globalErrorExitStatus)
+	}
+}
+
+func mainRemoveTag(ctx *cli.Context) error {
+	checkRemoveTagSyntax(ctx)
+	setTagSetColorScheme()
+	var pErr *probe.Error
+	objectURL := ctx.Args().Get(0)
+
+	alias, _ := url2Alias(objectURL)
```

Comment:
Is this check required. Won't the `newClient` function fail in that case?

Reply:
Removed in push.